conformance/Rakefile

```diff
@@ -36,6 +36,28 @@ namespace :tlds do
     File.open(repo_path('tld_lib.yml'), 'w') do |file|
       file.write(yml.to_yaml)
     end
+
+    File.open(repo_path("TldLists.java"), 'w') do |file|
+      file.write(<<-EOF
+package com.twitter;
+
+import java.util.Arrays;
+import java.util.List;
+
+// auto generated!
```

Comment:
imho, move the comment to the top of the file and change it to say:
// Auto-generated by conformance/Rakefile